bin/boot

```diff
@@ -83,17 +105,23 @@ function on_exit() {
 }
 trap on_exit INT TERM
 
-echo $log_src[`date +%F.%H:%M:%S`]' Search conf file'
-if [[ -z "$ODOO_CONF" ]];
-then
-    file="/opt/odoo/etc/odoo.conf"
-else
-    file=$ODOO_CONF
+echo $log_src[`date +%F.%H:%M:%S`]' Search for target user...'
+bash /opt/sources/target_user.sh
+
+if [[ -z "$TARGET_ID" ]]; then
+    echo $log_src[`date +%F.%H:%M:%S`]' No target user, falling back to odoo'
+    TARGET_ID=$( id -u odoo )
 fi
+
+echo $log_src[`date +%F.%H:%M:%S`]' Search conf file...'
+if [[ -z "$odoo_conf_file" ]]; then
+    odoo_conf_file="/opt/odoo/etc/odoo.conf"
+fi
+
 echo $log_src[`date +%F.%H:%M:%S`]' Running command...'
 for arg in "$*"
 do
     $arg
 done
 
-wait
+wait
```

Comment:
Empty line at the end of a file is a PEP8 convention so it only applies for `.py`